### ISSUE TITLE
Mini-cart: headline only changes to plural after page reload 

### ISSUE THREAD

**Issue by KFie** [Xceptance/posters-demo-store#129]
Reproduce: 
1. start out with empty cart 
2. add one item to cart 
3. add another item to cart
4. mouse over cart icon to trigger mini-cart flyout 

Issue: The headline reads "2 item in the cart", it fails to adjust to the correct plural. Only after page refresh do I see the correct version "2 items in the cart". Expected to see this right away. 

https://github.com/Xceptance/posters-demo-store/assets/141221329/65e43dcf-6667-4b78-9d6e-dacafb168734

![2023-08-04_11h07_56](https://github.com/Xceptance/posters-demo-store/assets/141221329/2cd0f497-cb92-46ca-bb5d-147a56aff067)

**Comment by freddyka** [comment]
Issue remains for poster v2.1.

**Comment by EliasTatX** [comment]
This is still present as of posters v2.2. A duplicate of this ticket has been created in #209
